### DISCUSSION TITLE
dark theme

### DISCUSSION THREAD

**Discussion by BidemiEnoch** [Tekipeps/aul-forum#29] [General]
Is there a need to add a dark theme, or the light version is just sufficient?

**Comment by Tekipeps** [comment]
Yes dark theme can be integrated easily with styled components.. we can do that later on
I feel we should wipe out sass completely if we choose to use styled components..

Also the dashboard page (after login) should be designed so we can integrate authentication,
I've finished all routes for authentication in backend so we can move to connecting with the frontend

**Reply by BidemiEnoch** [reply]
I thought about this, to my annoyance implementing dark themes in sass is less flexible since you can just pass down the theme as a prop with styled components, Very annoyed I just spent a whole day integrating sass modules!! Aight, ill remove sass as a dependency then, I should have made a fully functional dark/light theme toggler by tommorrow...maybe

**Reply by BidemiEnoch** [reply]
My idea was to redirect to home after login, well, until we connect the backend to frontend and start passing *actual data* to the components, I cant work on some features, I also need to break down a lot of components into smaller parts, Just waiting for when I can actually get some backend data

**Reply by BidemiEnoch** [reply]
on a final thought, im scrapping the dark  theme feature, a lot of components(esp images) require the light background to blend in.
sass stays ~~Smiles secretly cos sass is going no where~~, I could easily scrap styled components if you dont want use to use both
